src/main/util.ts

```diff
@@ -37,5 +39,17 @@ export const mkdir = (dirPath: string) => {
   }
 };
 
+export const handleOSQuery: () => OperatingSystems = () => {
```

Comment:
Maybe return an enum instead, as this will always (or at least should, assuming the user is on a supported OS) have exactly one 'true' value

Reply:
nice catch

enum refactor: https://github.com/patrickbrett/mlvet/pull/38/commits/f54cbe3641987ce328de2f5e4091eb81a6eb50d4